app/src/test/kotlin/integration/IntegrationTest.kt

```diff
@@ -0,0 +1,63 @@
+package integration
+
+import androidx.compose.foundation.interaction.HoverInteraction
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.material.*
+import androidx.compose.runtime.*
+import androidx.compose.ui.test.*
+import androidx.compose.ui.test.junit4.createComposeRule
+import kotlinx.coroutines.runBlocking
+import model.graphs.UndirectedGraph
+import org.junit.Rule
+import org.junit.Test
```

Comment:
Yep, i found out that only at the end, bc i couldn;t find any examples of UI testing in case of compose using the JUnit4, so i managed to delete the task , that caused problems with unit-tests)

Reply:
Btw, only recommendation i saw - to delete the task)